dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -37,18 +37,21 @@ type InlineProps = {
 	position: InlinePosition;
 	index: number;
 	shouldHideReaderRevenue?: never;
+	isDark?: never;
 };
 
 type RightProps = {
 	position: 'right';
 	index?: never;
 	shouldHideReaderRevenue: boolean;
+	isDark?: boolean;
```

Comment:
can we avoid needing this prop and make decisions based on article design please?

Reply:
as mentioned above as the label is only dark for the right ad slot only, I don't think adding a palette theme makes sense?